nemo/collections/tts/data/dataset.py

```diff
@@ -453,6 +456,13 @@ def add_pitch(self, **kwargs):
             with open(Path(pitch_stats_path), 'r', encoding="utf-8") as pitch_f:
                 self.pitch_stats = json.load(pitch_f)
 
+    # Add pretrained speaker embeddings (e.g. speaker-verification embeddings)
```

Comment:
Are we using titanet embeddings? Do we have a script to compute the speaker embeddings and store them like other data types in sup_data?